.github/workflows/sanitizers.yml

```diff
@@ -0,0 +1,170 @@
+# Check code with compilers' sanitizers
+name: Sanitizers
+
+on: workflow_call
+
+env:
+  BUILD_DIR : "${{github.workspace}}/build/"
+
+permissions:
+  contents: read
+
+jobs:
+  icx-build:
+    name: Intel C++ Compiler
```

Comment:
There is no naming convention now. Names doesn't relate to the same thing. Now we have three jobs:
- Intel C++ Compiler
- Ubuntu
- Windows
I suggest to stick with the naming based on the compilers used because that is the main differentiator between all of the jobs.

Reply:
I proposed a different name, what do you think? BTW, when we introduce dockers there will be again some changes in these names and jobs :wink